clusters/l3-sqnc/monitoring/kube-prometheus-stack/values.yaml

```diff
@@ -0,0 +1,109 @@
+alertmanager:
+  enabled: true
+  serviceMonitor:
+    selfMonitor: true
+    namespace: monitoring
+  config:
+    receivers:
+      - name: "null"
+      # - 
+      # - name: slack
+      #   webhook_configs:
+      #     - send_resolved: true
+      #       url: <secret>
+  alertmanagerSpec:
+    replicas: 2
+    podAntiAffinity: soft
+prometheus:
+  serviceMonitor:
+    selfMonitor: true
+    namespace: monitoring
+  service:
+    sessionAffinity: ClientIP # Ensures grafana always pulls from the same replica
+  prometheusSpec:
+    replicas: 2
+    podAntiAffinity: soft
+    retention: 12h
+    storageSpec:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: managed-csi-premium
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 50Gi
+    resources:
+      requests:
+        cpu: 200m
+        memory: 200Mi
+      limits:
+        cpu: 2000m
+        memory: 2Gi
+    serviceMonitorSelectorNilUsesHelmValues: false
+    serviceMonitorNamespaceSelector: { }
+    podMonitorSelectorNilUsesHelmValues: false
+    podMonitorNamespaceSelector: { }
+    podMonitorSelector:
+      matchLabels:
+        app.kubernetes.io/component: monitoring
+grafana:
+  defaultDashboardsEnabled: false
+  serviceMonitor:
+    enabled: true
+    namespace: monitoring
+  ingress:
+    enabled: true
```

Comment:
At the moment I assume grafana will be accessed using the admin password (which I presume you have somewhere?). I know the plan is to link this to our Okta SSO, is it worth disabling the ingress for now until we have that setup?

Reply:
Grafana will be accessed via the admin password (In the secret we added in this PR).  It is fine to have the ingress enabled.

Reply:
Can I understand why you think that? I get it's secured in some way, but password sharing for operational activities is definitely a no no.

I'm only suggesting this not be on the public facing internet until we have auth properly in place

Reply:
Only a limited subsection of the development team are capable of retrieving this secret, the same subset are the only ones capable of port-forwarding to the grafana service itself.